virttest/shared/deps/rss/setuprss.bat

```diff
@@ -1,5 +1,18 @@
+rem Enable delayed expansion to be able to use !ERRORLEVEL!
+setlocal enabledelayedexpansion
 set rsspath=%1
-if [%1]==[] set rsspath=%~dp0\rss.exe
+
+if [%1]==[] (
+    echo "Guessing rss path to use based on processor architecture"
+    reg query "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Environment" /v PROCESSOR_ARCHITECTURE | find /i "x86" > nul
```

Comment:
When searching for ways to check the architecture I stumbled upon [this comment](https://msfn.org/board/topic/129674-how-can-i-identify-if-os-is-32-bit-or-64-bit-in-batch-file/?do=findComment&comment=1089306):

> ```bat
> SET "ARCH=x64"IF /I "%PROCESSOR_ARCHITECTURE%"=="x86" (    IF NOT DEFINED PROCESSOR_ARCHITEW6432 SET "ARCH=x86")
> ```
> 
> But there have been problems reported even with this method. There are situations in Win7 install where the eventual install will be x64 but the environment variables, and the file system at that point, are in a 32bit state so %PROCESSOR_ARCHITECTURE% will return x86 on AMD64 hardware, (example bug over at DriverPacks.net: http://forum.driverpacks.net/viewtopic.php?pid=48416#p48416), and SysWOW64 might not exist yet, though most of the time just checking for SysWOW64 will work here.

Since this bat file is called during the unattended installation, I end up choosing the option suggested by the same person a few lines above the quote:
> That is why MS suggests not relying on any file location at all but rather the values of system variables. A reliable result for OS bitness can be obtained by checking PROCESSOR_ARCHITECTURE and PROCESSOR_ARCHITEW6432 without checking folders or files, provided that nobody changed either value in the registry at "HKLM\SYSTEM\CurrentControlSet\Control\Session Manager\Environment".

But I don't have objections against using the `PROCESSOR_ARCHITECTURE` variable, if you happen to prefer it instead.

Reply:
ok, thanks for your explanation.

Reply:
Thanks for the review, @luckyh 
For future readers, I have added a comment to the bat script explaining why the variable is not used directly.